consensus/impl/dpos/slot/slot.go

```diff
@@ -136,7 +136,7 @@ func (s *Slot) NextBpIndex(bpCount uint16) int64 {
 
 // BpMaxTime returns the max time limit for block production in nsec.
 func BpMaxTime() time.Duration {
-	return time.Duration(bpMaxTimeLimitMs) * 1000
+	return time.Duration(bpMaxTimeLimitMs) * time.Millisecond
```

Comment:
Is this correct? The comment says "nsec" and the function uses `time.Millisecond`...